test/python/tensorflow/tests_linux_ie_cpu.txt

```diff
@@ -44,6 +44,10 @@ argmax_op_test.ArgMaxTest.testOutputEmpty
 argmax_op_test.ArgMaxTest.testEmpty
 array_ops_test.SnapshotOpTest.testInvertPermutation
 array_ops_test.StridedSliceGradTest.testInt64Indices
+array_ops_test.StridedSliceTest.testExpand
```

Comment:
Can we put some comments as to why these are skipped?

Reply:
I think these are enabled now with the change

Reply:
That's correct.